node-install.sh

```diff
@@ -1,6 +1,59 @@
 #!/bin/bash
 
-echo 'Node.js Linux Installer by github.com/taaem, updated by github.com/MichaIng'
+show_help() {
+    cat <<END
+Universal Node.js installer for Linux. Original project by taaem (github.com/taaem/nodejs-linux-installer) and 
+    MichaIng (github.com/MichaIng/nodejs-linux-installer).
+
+Use this script to install latest version available for your CPU architecture.
+
+Usage: node-install.sh [<params>]
+
+Parameters:
+    -lu
+    --list-unofficial-releases          List all available versions in nodejs.org's unofficial builds.
+    
+    -u <version>
```

Comment:
I really like the idea of using latest as default, when no version is given.
Is this, amongst others, what you refer to by adding some more command line options? Otherwise I can offer to work on resolving the latest unofficial version for the given architecture. That's if you can spare the extra time me writing the code instead of you.

Reply:
I just recognised the `sort -V` flag, which makes obtaining the latest version from the list trivial 👍.

I would merge the PR as it is now (after testing the next hours) and we can implement further enhancements with a separate one. To keep it as simple and intuitive as possible:

- Install latest official build available for the architecture by default. But switch to the cleaner method via `index.tab` for official builds as well.
- Use `-V | --version` (or `-v`, but more conflicting with standard arguments?) to optionally define a specific version, rather than the latest.
- Use `-u | --unofficial` to include unofficial builds. We can scrape both `index.tab` lists, sorting out the lines where field 3 does not contain the own architecture, compare and use the newest respectively matching version of both lists. If in rare circumstances both lists contain the same version for the own architecture (probably the case for older `armv6l` versions), the official build is preferred.
- Use `-l | --list` to list all versions available for the own architecture, combine with `-u` (including `-lu`,`-ul` and `--list --unofficial`) to print both lists, for unofficial builds as well. The function to scrape and print the lists can be used to get the latest version as well. When printing to console, we probably find a way to nicely print the lists side by side or one line (+ another line for unofficial builds) to the terminal, to avoid extensive scrolling but have the the latest version at the top-left?

Reply:
Lol and now I see your commit 😄.